README.md

```diff
@@ -66,6 +66,7 @@ Input                     | Description
 `test-results-simulink-test` | (Optional) Path to export Simulink Test Manager results in MLDATX format. This input requires a Simulink Test license, and is supported in MATLAB R2019a and later.<br/>**Example:** `test-results/results.mldatx`
 `code-coverage-cobertura`   | (Optional) Path to write code coverage report in Cobertura XML format.<br/>**Example:** `code-coverage/coverage.xml`
 `model-coverage-cobertura`  | (Optional) Path to write model coverage report in Cobertura XML format. This input requires a Simulink Coverage™ license, and is supported in MATLAB R2018b and later.<br/>**Example:** `model-coverage/coverage.xml`
+`use-parallel`              | (Optional) Indicator to run tests in parallel, specified as false or true. This input requires a Parallel Computing Toolbox license, and might not be compatible with other options. If incompatible, testing occurs in serial regardless of the value for `use-parallel`.
```

Comment:
This needs a note here and potentially also down in `action.yml` saying that this option requires a self-hosted runner as current licensing restrictions on GitHub-hosted runners do not allow opening a parallel pool. @mw-hrastega any ideas?

Reply:
A few comments:

1) Should this option come last in the table? Does it make sense to have it before or after the artifact-related options?
2) Which options (in the table) is `use-parallel` not compatible with?
3) I would propose changes to the text as follows:

(Optional) Whether to run tests in parallel on a self-hosted runner, specified as `false` or `true`. By default, the value is `false` and tests run in serial. When the value is `true`, tests run in parallel. This input requires a Parallel Computing Toolbox&trade; license, and is supported only on self-hosted runners.

Reply:
1. Good question. We currently have 3 logically grouped set of options. Options to: filter tests, generate test artifacts, and generate coverage artifacts. `source-folder` and `use-parallel` are outliers because they don't fit under any of these groups. Perhaps `use-parallel` should move up right below `source-folder` as a forth logical grouped set of "general" options.
2. That is a complex topic. Compatibility depends on release. For example, you can produce JUnit results in parallel in R2018b but you can't produce Cobertura model coverage in parallel until R2021b. We always fall back to running in serial if the user requests an artifact that cannot be produced in parallel with their current release of MATLAB.
3. OK

Reply:
1. Perhaps. You can take a look at how we have recently categorized name-value arguments for runtests.

3.  Should we say something like this?

(Optional) Whether to run tests in parallel on a self-hosted runner, specified as `false` or `true`. By default, the value is `false` and tests run in serial. If the test runner configuration is suited for parallelization, a `true` value results in tests being run in parallel. This input requires a Parallel Computing Toolbox™ license, and is supported only on self-hosted runners.

Reply:
Updated `use-parallel` description and order per feedback.